src/pages/index.js

```diff
@@ -70,17 +78,31 @@ import { PopupWithImage } from "../components/PopupWithImage";
     errorClass: "popup__error_visible",
   };
 
+  const avatarConfiguration = {
+    formSelector: ".modal-avatar__container",
+    inputSelector: ".modal-avatar__input",
+    submitButtonSelector: ".modal__save-button",
+    inactiveButtonClass: "transparent",
+    nputErrorClass: "popup__input_type_error",
+    errorClass: "popup__error_visible",
+  }
+
   const editValidator = new FormValidator(configuration, editForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```